src/main/java/dev/emi/trinkets/TrinketsMain.java

```diff
@@ -106,9 +106,33 @@ public void onInitialize() {
 						)
 					)
 				)
+				.then(
+					literal("clear")
+					.executes(context -> {
+						try {
+							return clearCommand(context);
+						} catch (Exception e){
+							e.printStackTrace();
+							return -1;
+						}
+					})
+				)
 			));
 	}
 
+	private static int clearCommand(CommandContext<ServerCommandSource> context){
+		ServerPlayerEntity player = context.getSource().getPlayer();
+		if (player != null) {
+			TrinketComponent comp = TrinketsApi.getTrinketComponent(player).get();
+			for (var entry : comp.getInventory().entrySet()){
```

Comment:
No `var` please, makes it hard to review and maintain. I'll note this in further comments as well

Reply:
Can remove. Its not the only var in your codebase fyi. Can track the others down too.